src/formpack/schema/fields.py

```diff
@@ -560,6 +621,96 @@ def get_labels(self, *args, **kwargs):
         return [self.name]
 
 
+class IdCopyField(CopyField):
+
+    FIELD_NAME = '_id'
+
+    def __init__(self, section=None, *args, **kwargs):
+        super(IdCopyField, self).__init__(
```

Comment:
We can drop Python2 support. ( I though we did :-) )

Reply:
It seems it was only in my dreams. 
https://github.com/kobotoolbox/formpack/issues/239